base/applications/taskmgr/perfdata.h

```diff
@@ -33,6 +33,8 @@ typedef struct _PERFDATA
 	WCHAR				ImageName[MAX_PATH];
 	HANDLE				ProcessId;
 	WCHAR				UserName[MAX_PATH];
+// TODO: Store actual length, to avoid wcslen() on CachedGetUserFromSid() cache hit.
+//	ULONG				UserNameLength;
```

Comment:
This is not necessary unless someone finds that it becomes a bottleneck.

Reply:
I think TaskMgr, running continuously, should use a little resource as possible. Anyway.
Reverted.